doc/devel/documenting_mpl.rst

```diff
@@ -19,6 +24,31 @@ The requirements are as follows (https://github.com/matplotlib/matplotlib/blob/m
 6. pillow
 7. graphviz
 
+.. note::
+
+  * You'll need a minimal working LaTeX distribution for many examples to run.
+  * `Graphviz <http://www.graphviz.org/Download.php>`_ is not pip-installable so you need
+    to install this on your own.
```

Comment:
To me this sounds like Graphviz is a python package which just happens to be non pip installable. I think it could be made more clear that it is a completely separate binary and not a python package.